src/LoveMachine.Core/Config/OscillateConfig.cs

```diff
@@ -0,0 +1,34 @@
+﻿using BepInEx;
+using BepInEx.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace LoveMachine.Core.Config
+{
+    internal static class OscillateConfig
+    {
+        public static ConfigEntry<ConstrictMode> Mode { get; private set; }
+
+        internal static void Initialize(BaseUnityPlugin plugin)
+        {
+            int order = 1000;
+            const string constrictSettingsTitle = "Oscillate Settings";
+            Mode = plugin.Config.Bind(
+               section: constrictSettingsTitle,
+               key: "Oscillate Mode",
+               defaultValue: ConstrictMode.Cycle,
```

Comment:
Would it make more sense to make speed the default?
I don't have an oscillating device but I'd imagine one would want to more or less match the oscillation speed to the in-game speed.

Reply:
Yes that was what I wanted to do, but i failed ^^